lib/libprocstat/libprocstat.c

```diff
@@ -479,7 +479,6 @@ procstat_getcompartments(struct procstat *procstat, struct kinfo_proc *kp,
 out_free:
```

Comment:
Note that if you initialized `inbuf` to `NULL` at the start of the function you could consolidate this down to a single label (but as a followup commit if you wanted to do that).